src/items/use-declarations.md

```diff
@@ -82,31 +82,44 @@ In this example, the module `quux` re-exports two public names defined in
 
 ## `use` Paths
 
-Paths in `use` items must start with a crate name or one of the [path
-qualifiers] `crate`, `self`, `super`, or `::`. `crate` refers to the current
-crate. `self` refers to the current module. `super` refers to the parent
-module. `::` can be used to explicitly refer to a crate, requiring an extern
-crate name to follow.
+Paths in `use` items must start with an identifier or one of the [path
+qualifiers] `crate`, `self`, `super`, or `::`.
 
-An example of what will and will not work for `use` items:
-<!-- Note: This example works as-is in either 2015 or 2018. -->
+`crate` refers to the current crate. `self` refers to the current module.
+`super` refers to the parent module. `::` refers to the [extern prelude] (so
+it requires a crate name to follow).
 
-```rust
+If the path starts with an identifier, the identifier is resolved in the same
+way as for a path in an expression (in particular, it can be a crate name or
```

Comment:
"Expression" here is a bit too specific, this is true for any non-import path.

Reply:
>If the path starts with an identifier

And if the path doesn't start with an identifier, the behavior is also the same.
The text leaves the non-identifier case up in the air though and never talks about it below.